src/rules.ts

```diff
@@ -588,23 +589,12 @@ class GitTrailerContainsWhitespace implements IConventionalCommitRule {
 }
 
 /**
- * Footer should not contain any blank line(s)
+ * Rule C022 was historically known as:
+ *     FooterContainsBlankLine
+ * with description:
+ *     "Footer should not contain any blank line(s)";
+ * This rule has been removed and its ID should therefore not be re-used.
  */
-class FooterContainsBlankLine implements IConventionalCommitRule {
```

Comment:
Rule is no longer valid, as we now support cutting-lines.
As a side-effect, all git-trailers can now include blank lines.

Reply:
What happens if someone have this rule configured in their configuration file explicitly? Depending on the answer this removal might be a breaking change in fact.

Reply:
It will raise a `warning` in the logging but continue its execution